src/styles/components/layout.scss

```diff
@@ -1,23 +1,79 @@
 @import "./../settings.scss";
 @import "./../tools.scss";
-
 .layout {
-  display: flex;
-
-  .header {
     display: flex;
-    flex-direction: column;
-    position: fixed !important;
-    top: 0 !important;
-    left: 0;
-    width: 100%;
-    margin: 0;
-    min-height: base-spacing(9);
-    background: $almos-black;
-    z-index: 25 !important;
-  }
-
-  .footer {
-    display: flex;
-  }
-}
+    .header {
+        display: flex;
+        flex-direction: row;
+        position: fixed !important;
+        top: 0 !important;
+        left: 0;
+        width: 100%;
+        margin: 0;
+        min-height: base-spacing(7);
+        background: $almos-black;
+        z-index: 25 !important;
+    }
+    .footer {
+        display: flex;
+    }
+    .header_logo {
+        width: 330px;
+        padding-top: 10px;
+        padding-left: 170px;
+    }
+    .header_logo img {
+        height: 37px;
+    }
+    .menu {
+        width: 500px;
+        height: 20px;
+        display: flex;
+    }
+    .menu a {
```

Comment:
You shouldn't use a selector which based on elements better create a class for a elements.
It makes your competent independent because you can use this calls for any element